sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_feature_set/featureset_spec_properties_schema.py

```diff
@@ -38,6 +38,5 @@ class FeaturesetSpecPropertiesSchema(YamlFileSchema):
         FeatureTransformationCodePropertiesSchema, data_key="featureTransformationCode"
     )
     features = fields.List(NestedField(FeaturePropertiesSchema), data_key="features")
-    index_columns = fields.List(NestedField(DataColumnSchema), data_key="indexColumns")
```

Comment:
Do we set unknown=Exclude so that this doesn't throw?

Reply:
I verified, it is not throwing error since we are using it only for serialization

Reply:
When we load featureset.yaml from CLI, it will also load the spec from yaml right? Will it also work if we deserialize?